### DISCUSSION TITLE
Places to practice my C++ skills

### DISCUSSION THREAD

**Discussion by ghost** [community/community#106647] [Programming Help]
### Body

Hi, 

I am a terrible C++ programmer and I wanted to know how I could somehow get better at it even though it is extremely unlikely.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by DylanBT928** [comment]
[LearnCPP](https://www.learncpp.com/) is a great way to start if you're the reading type. If you are into videos, you could start by watching freeCodeCamp's [C++ Tutorial for Beginners](https://www.youtube.com/watch?v=vLnPwxZdW4Y&pp=ygUQZnJlZWNvZGVjYW1wIGMrKw%3D%3D) and they also have a [Beginner to Advanced](https://www.youtube.com/watch?v=8jLOx1hD3_o&t=20s) course. These three resources are free and widely used by others. 

Hoped this helped, have fun learning!

**Comment by MohammedAbrarAli** [comment]
There are few platforms that are good for practice 
Here are some 
HackerRank
W3school
Geeksforgeeks
Consistency is mandatory for learning anything

**Comment by GaneshTamang** [comment]
one of the best i have found are
* w3 schools
* leetcode
* geekforgeeks

**Comment by Shahseo** [comment]
To improve in C++, focus on practicing regularly, utilizing online resources, working on projects, seeking help when needed, reviewing and refactoring code, and staying persistent and patient with yourself.
